src/views/GameRoom/components/GameTable/styles.ts

```diff
@@ -1,18 +1,53 @@
 import styled from 'styled-components';
+import { CardWrapper } from '../../../../components/PlayingCard/styles';
 
 export const TableWrapper = styled.div`
-  background: #cfd8dc;
+  background: #272627;
+  height: 80vh;
+
   display: grid;
+  grid-gap: 16px;
   grid-template-columns: repeat(5, 1fr);
   grid-template-rows: repeat(4, 1fr);
   grid-template-areas:
-    'blank player player player blank'
-    'player blank active deck player'
-    'player blank blank blank player'
-    'blank player you player blank';
-  padding: 8px;
+    'corner-1 player-4  player-5  player-6  corner-2'
+    'player-3 center    center    center    player-7'
+    'player-2 center    center    center    player-8'
+    'coner-4  player-1  player-0  player-9  corner-3';
```

Comment:
You son of a bitch.

Reply:
I'm in

Reply:
This gonna be we awesome when doing mobile version

Reply:
Good job Kevin!